doc/getting_started/starting.rst

```diff
@@ -181,7 +181,7 @@ For clarity, we'll leave out the data and make a nicely-formatted output:
 
 Notice here that ``encode(x='a')`` has been expanded to a JSON structure with
 a ``field`` name, and a ``type`` for the data.
-The ``encode(y='b')`` has been expanded similarly and includes an ``aggregate``
+The ``encode(y='average(b)')`` has been expanded similarly and includes an ``aggregate``
```

Comment:
My intent in this line was to mention how the previous ``encode(y='b')`` is now changed to ``encode(y='average(b)'`` which includes an aggregate... seems that was unclear, but I don't think this change conveys the correct idea either.